### DISCUSSION TITLE
Использование Mockoon для внешней сети

### DISCUSSION THREAD

**Discussion by latinos008** [mockoon/mockoon#925] [General]
Доброго времени суток! Я наткнулся на программное обеспечение mockoon, и не могу понять, может ли оно помочь решить мою проблему? У меня есть свой сайт, внутри которого в Iframe отображается другой сайт, к которому у меня нет доступа. В свою очередь чужой сайт посылает get, post запросы на свой сервер, с которого получает ответы. Вопрос заключается в том, чтобы перехватить один из ответов и изменить тело ответа. Так, чтобы это отображалось у посетителей моего сайта, Подскажите, такое может реализовать mockoon?

**Comment by 255kb** [comment]
Mockoon cannot intercept requests, especially in a production environment. You may be more lucky with a software like Wireshark
